### PR TITLE
ci: enable more tests

### PR DESCRIPTION
**Pull request by EmilienM**
More tests could be run, let's just enable them in the e2e manifest.

### CODE REVIEW

File: test/e2e/manifest.yaml [hunk 1]
```diff
@@ -14,3 +14,12 @@ DriverInfo:
   SupportedSizeRange:
     Min: 1Gi
     Max: 16Ti
+  Capabilities:
+    persistence: true
+    # our CI uses CephFS / NFV driver, which doesn't support fsgroups.
```
Comment (gouthampacha) [inline test/e2e/manifest.yaml:19]: NFS*

File: test/e2e/manifest.yaml [hunk 2]
```diff
@@ -14,3 +14,12 @@ DriverInfo:
...
+    fsGroup: false
+    exec: true
+    # our CI uses CephFS / NFV driver, which doesn't support snapshots.
```
Comment (gouthampacha) [inline test/e2e/manifest.yaml:22]: NFS*

clarifying detail: The driver supports snapshots in the wallaby release of OpenStack Manila with Ceph Octopus+;

**Review by gouthampacha — APPROVED** [review]
Minor typo in the comments, but otherwise this looks great. Thanks @EmilienM

**Comment by gouthampacha** [comment]
Thanks, LGTM! :)

**Comment by mandre** [comment]
/lgtm

For the curious ones, we went from running [234](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_csi-driver-manila-operator/137/pull-ci-openshift-csi-driver-manila-operator-master-e2e-openstack-csi/1487440300606492672) tests to [244](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_csi-driver-manila-operator/138/pull-ci-openshift-csi-driver-manila-operator-master-e2e-openstack-csi/1497050961636495360).

**Pull request merged**